charts/eks-anywhere-packages/templates/cronjob.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  namespace: {{ .Values.namespace }}
```

Comment:
I almost wonder if these secrets and this cron job should run in the eksa-system namespace instead of eksa-packages

Reply:
The reasoning for running this in eksa-packages is the configmap that keeps track of namespaces that the controller installs to.